Corona-Warn-App/src/main/res/values/legal_strings.xml

```diff
@@ -146,4 +146,22 @@
     <string name="trace_location_onboarding_body_consent5" translatable="false">If you later use the warning feature yourself, all of the other users who were checked in at the same time as you at an event or place listed under “My check-ins” will receive a warning based on the event ID.</string>
     <!-- YMSG: Onboarding trace location bullet points -->
     <string name="trace_location_onboarding_body_consent6" translatable="false">You can erase a check-in at any time under “My check-ins”. In this case, you will only be warned if your smartphone identifies a possible exposure after receiving the random IDs of the user who tested positive.</string>
+
+    <!-- RAT Profile Onboarding -->
+    <!-- XHED: Title for RAT Profile onboarding consent card -->
+    <string name="rat_profile_onboarding_card_title" translatable="false">Data protection and data security</string>
+    <!-- YMSG: RAT Profile Onboarding bullet points -->
+    <string name="rat_profile_onboarding_card_consent1" translatable="false">Creating a rapid test profile in the app, and using it, are voluntary.</string>
+    <!-- YMSG: RAT Profile Onboarding bullet points -->
+    <string name="rat_profile_onboarding_card_consent2" translatable="false">The app converts the data you enter for your rapid test profile into your personal QR code. The QR code then contains all the data you entered.</string>
+    <!-- YMSG: RAT Profile Onboarding bullet points -->
+    <string name="rat_profile_onboarding_card_consent3" translatable="false">At a testing point, you can then have your rapid test profile’s QR code scanned as a means of quickly and securely providing your data to the testing point. The testing point will need your data to carry out the test. However, you do not need the app to provide your data. You decide yourself which data is included in the QR code, and which information you communicate to the testing point by other means.</string>
+    <!-- YMSG: RAT Profile Onboarding bullet points -->
+    <string name="rat_profile_onboarding_card_consent4" translatable="false">To begin with, only you will have access to your personal rapid test profile. Other people can only access the personal data contained in your rapid test profile if you show them your rapid test profile’s QR code. This will not provide access to other data from the app (e.g. your check-ins or information in your contact journal).</string>
+    <!-- YMSG: RAT Profile Onboarding bullet points -->
+    <string name="rat_profile_onboarding_card_consent5" translatable="false">You have the possibility to delete your rapid test profile at any time. Until then, it will remain stored in the app on your device.</string>
```

Comment:
Hi, could you please change 'device' - to 'smartphone'? Thanks.
```suggestion
    <string name="rat_profile_onboarding_card_consent5" translatable="false">You have the possibility to delete your rapid test profile at any time. Until then, it will remain stored in the app on your smartphone.</string>
```